public/app/features/logs/components/logParser.ts

```diff
@@ -57,10 +57,6 @@ function shouldRemoveField(field: Field, index: number, row: LogRowModel) {
   if (field.name === 'id' || field.name === 'tsNs') {
     return true;
   }
-  // entry field which we are showing as the log message
```

Comment:
Maybe we could hide the line when there are no links? In theory (but not the cleanest way), it's possible to do something like:

```typescript
  if (row.entryFieldIndex === index && getFieldLinks && getFieldLinks(field, row.rowIndex, dataFrame).length === 0) {
    return true;
  }
```

Reply:
Ah crap I forgot to push the change to LogDetails.tsx that does this - can you check out that change above?